sharedcount.php

```diff
@@ -34,8 +34,77 @@ class SharedCount {
 		static function init() {
 			$c = get_called_class();
 
+			register_activation_hook( __FILE__, array( $c, 'sharedcount_activate' ) );
```

Comment:
Instead of using an activation hook to register the cronjob, just check if the event is registered, and if not, register it.  This way if some other module messes with the cron job, it will be automatically re-scheduled.